src/Context/NoteContext.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import { createContext, useContext } from "react";
+import { useState } from "react";
+import axios from "axios";
+import { useEffect } from "react";
+const NoteContext = createContext()
+
+const useNote = () => useContext(NoteContext)
+
+const initialState = {
+    title:"",
+    content:"",
+    priority:"",
+}
+const NoteProvider = ({children}) =>{
+    const [note, setNote] = useState(initialState)
+    const [archive, setArchive] = useState([])
+    const token = localStorage.getItem("token")
```

Comment:
Don't you need to first check if the token exists?

Reply:
After login then only user can add note I have made different pages for them.